src/config/sendMail.js

```diff
@@ -0,0 +1,28 @@
+// Importing the configured transporter object from the mailTrap.js
+const transporter = require('./mailTrap');
+
+// Function to send an email using the configured transporter object
+function sendEmail(toName, toEmail, subject, text) {
+  // Creating mail options with correct recipient format, subject, and text body 
+  const mailOptions = {
+    // Sender email address
+    from: '',
+    // Recipient email address
+    to: `${toName} <${toEmail}>`,
+    // Email subject
+    subject: subject,
+    // Email body text
+    text: text
+  };
+
+  // send the email using the transporter 
+  transporter.sendMail(mailOptions, (error, info) => {
+    // return errors, if any
+    if (error) {
+      return error;
+    }
+  });
+}
```

Comment:
kindly share some insight on the info parameter, i see the error being return but on success nothing is done. I will like to know on success is the email still being sent, what is the role of the info please? Kindly reply to this comment i find wrong with script.